package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zowe-cli-cics-deploy-plugin",
-  "version": "0.5.0",
+  "version": "1.0.0",
```

Comment:
I think we should leave this at 0.5 for now so we can do a test publish

Reply:
done

Reply:
Actually.... do we need to update this to 0.6 in order to do the test publish, sorry hadn't thought that through!